src/twitch.js

```diff
@@ -9,8 +9,13 @@ const Twitch = {
         return GraphQL.SendQuery("GET_USER", variables);
     },
     GetTopStreams(amount = 25, variables = {}) {
-        variables = {after: "", ...variables, amount};
-        return GraphQL.SendQuery("GET_TOP_STREAMS", variables);
+        variables = {
+            ...variables,
+            limit: amount,
+            platformType: "all",
+            sortTypeIsRecency: false
```

Comment:
I'd suggest putting `...variables` after this in the rare case you'd want to overwrite them in your call.

Reply:
I left it because it was a queries before, it's better to be able to overwrite it, indeed